docs/capability-types.md

```diff
@@ -0,0 +1,130 @@
+This is a full list of capability types with their properties, entities and units. See the [general information about capabilities](fixture-format.md#capabilities).
+
+### Possible entities and keywords
+
+| Entity | Allowed Units | `-100%` | `-1%` | `0%` | `1%` | `100%`
+| - | - | - | - | - | - | -
+| Speed | `Hz`, `%` | fast reverse | slow reverse | stop | slow | fast
+| RotationSpeed | `Hz`, `%` | fast CCW | slow CW | stop | slow CW | fast CWW
+| Time | `s`, `ms`, `%` | – | – | instant | short | long
+| Distance | `m`, `%` | – | – | – | near | far
+| Brightness | `lm`, `%` | – | – | off | dark | bright
+| ColorTemperature | `K`, `%` | – | – | – | warm | cold
+| FogOutput | `m³/min`, `%` | – | – | off | weak | strong
+| RotationAngle | `°`, `%` | – | – | – | – | –
+| BeamAngle | `°`, `%` | – | – | closed | narrow | wide
+| Factor | (no unit), `%` | – | – | off | low | high
+| Index | (no unit) | – | – | – | – | –
+| Percent | `%` | – | – | off | low | high
+| Insertion | `%` | – | – | out | – | in
+| IrisPercent | `%` | – | – | closed | – | open
+
+### Possible capability types
+
+Required properties are _italic_, the entity is written in (parentheses).
+
+* Nothing
+* ShutterOpen
+* ShutterClosed
+* StrobeEffect
```

Comment:
Why not add `speed` and `time` (only one can be set at a time) here and remove `strobeType` from `StrobeSpeed` and `StrobeDuration`?

That'd remove the duplication of those keywords.

Reply:
That's true. Possible solution (similar to Effect[Speed/Intensity]):

* ShutterStrobe
  * _effect_: one of `Open`, `Closed`, `Strobe`, `StrobeRandom`, `Pulse`, `PulseRandom`, `RampUp`, `RampUpRandom`, `RampDown`, `RampDownRandom`
  * speed (Speed) (can't be used together with time)
  * time (Time) (can't be used together with speed)
* StrobeSpeed (global, doesn't activate strobe directly)
  * _speed_ (Speed)
* StrobeDuration (global, doesn't activate strobe directly)
  * _time_ (Time)

Reply:
Done.